test/metabase/upload_test.clj

```diff
@@ -1223,19 +1223,28 @@
   (mt/test-drivers (mt/normal-drivers-with-feature :uploads)
     (with-uploads-allowed
       (testing "Append should fail if there are extra or missing columns in the CSV file"
-        (doseq [csv-rows [["_mb_row_id,name,extra_column" "2,Luke Skywalker,so extra"] ;; extra column
-                          ["_mb_row_id" "2,Luke Skywalker"]                            ;; missing column
-                          ["_mb_row_id,extra_column" "2,so extra"]]]                   ;; extra and missing column
+        (doseq [[csv-rows error-message]
+                {["_mb_row_id,id,name,extra column one,EXTRA COLUMN TWO"]
+                 "The CSV file contains extra columns that are not in the table: \"extra_column_two\", \"extra_column_one\"."
+
+                 [""]
+                 "The CSV file contains extra columns that are not in the table: \"id\", \"name\"."
+
+                 ["_mb_row_id,extra 1, extra 2"]
+                 "The CSV file contains extra columns that are not in the table: \"extra_2\", \"extra_1\". The CSV file is missing columns that are in the table: \"id\", \"name\"."}]
```

Comment:
@luizarakaki can you review these error messages and make suggested changes if necessary?

Reply:
Yeah, these make sense!